config/tsconfig.build.json

```diff
@@ -9,7 +9,7 @@
     "inlineSources": false,
     "importHelpers": true,
     "lib": ["esnext", "dom"],
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
This is probably going to fuck-shit-up

Reply:
I should re-validate if I actually need this

Reply:
why did you need to update the module? did you run into some issues?

Reply:
Yeah it was building a esnext module, which node cannot run

Reply:
```
18:25:24 webpack | [webpack-cli] Failed to load '/Users/priscila/Projects/sentry/webpack.config.ts' config
18:25:24 webpack | [webpack-cli] /Users/priscila/Projects/sentry/webpack.config.ts:4
18:25:24 webpack | import { __assign, __read, __spreadArray } from "tslib";
18:25:24 webpack | ^^^^^^
```

Reply:
just saw it

Reply:
😢

Reply:
Good news is this seems to be a fine change. Everything is building. Just not getsentry yet

Reply:
I checked it briefly, but it seems that  we will have to sticky with commonjs 

...still want to read this:

https://github.com/TypeStrong/ts-node/issues/1007

Reply:
I checked it briefly and it seems we will have to sticky with `commonjs`